### ISSUE TITLE
Make "Select a Disc" dialog compulsory so that the selected disk is apparent

### ISSUE THREAD

**Issue by ozzdemir** [sysrich/tik#43]
First of all, thanks for the great work. I've been using Aeon from its iso times and wanted to make a new installation to a external SSD as a moveable OS. That's said, the following is what I run into.

I wanted to create a new Aeon install (RC3) on a external SSD of mine, following the guide [here](https://en.opensuse.org/Portal:Aeon/InstallGuide). Even though the external SSD and other media was attached, no "Select a Disc" dialog with no information on the selected disk being apparent I've proceeded without making sure the selected disk is indeed the external one hoping that the dialog will be eventually coming, but it was not the case. I did abort the process as soon as it started the deployment without selecting the disk and currently working on somehow recovering the corrupted disk.

I wanted to open this issue to make issues like this are less likely.

**Comment by sysrich** [comment]
No. We will only show the select a disc dialog when there is more than one disk to select.
